components/browser/icons/build.gradle

```diff
@@ -39,6 +43,16 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
     ]
 }
 
+tasks.register("updateExtensionVersion", Copy) {
+    from 'src/main/assets/extensions/browser-icons'
+    include 'manifest.json.template'
+    rename { 'manifest.json' }
+    into 'src/main/assets/extensions/browser-icons'
```

Comment:
I wonder if there's a target where we can put it and gradle picks it up - without it being in the src directory? Where do other code generators put there things? But not super important. :)

Reply:
I tried. And failed :). Couldn't figure out how to do this, but then this approach is also less magical. At least it's clear looking at the source that it's a template that will be replaced at build-time. Happy to revisit though.

Reply:
Alright, then let's go ahead as-is :)